extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -207,16 +203,15 @@ export class AmpA4A extends AMP.BaseElement {
 
   /** @override */
   renderOutsideViewport() {
-    // Only relevant if non-AMP as AMP creative will be injected within
-    // buildCallback promise chain.
-    // If another ad is currently loading we only load ads that are currently
-    // in viewport.
-    const allowRender = allowRenderOutsideViewport(this.element, this.win);
-    if (allowRender !== true) {
-      return allowRender;
+    // Ensure non-verified AMP creatives are throttled.
+    if (!this.isVerifiedAmpCreative_ && is3pThrottled(this.win)) {
+      this.emitLifecycleEvent('throttled3p');
+      return false;
     }
     // Otherwise the ad is good to go.
-    return super.renderOutsideViewport();
+    const elementCheck = getAmpAdRenderOutsideViewport(this.element);
+    return typeof elementCheck == 'number' ?
```

Comment:
to avoid this weird `typeof` check, shall we just make the function always return a number?